### PR TITLE
Add OpenAPI specification for apps bandwidth usage endpoints

### PR DESCRIPTION
**Pull request by ZachEddy**
Updates OpenAPI specification with the new apps bandwidth usage-related endpoints:

```
POST /v2/apps/metrics/bandwidth_daily
GET  /v2/apps/{app_id}/metrics/bandwidth_daily
```

### CODE REVIEW

**Review by jcodybaker — APPROVED** [review]

File: specification/resources/apps/examples/curl/apps_list_metrics_bandwidth_usage.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+lang: cURL
+source: |-
+  curl -X POST \
+    -H "Content-Type: application/json" \
+    -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
```
Comment (kamaln7) [inline specification/resources/apps/examples/curl/apps_list_metrics_bandwidth_usage.yml:5]:
```suggestion
    -H "Authorization: Bearer $DIGITALOCEAN_TOKEN" \
    -d '{ "app_ids": ["3aa4d20e-5527-4c00-b496-601fbd22520a"] }' \
```

**Review by kamaln7 — DISMISSED** [review]
LGTM!

File: specification/resources/apps/examples/curl/apps_list_metrics_bandwidth_usage.yml (hunk 1 above)
Comment (ZachEddy) [reply specification/resources/apps/examples/curl/apps_list_metrics_bandwidth_usage.yml:5]: Done 🙇

**Review by scotchneat — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**